src/agents/common.jl

```diff
@@ -38,209 +36,197 @@ end
 # TODO: Rethink file prefixes to create shorter directory names
 #       If do this, might need to be able to provide mapping function when doing integration testing
 function get_file_prefix(options::HEMOptions)
-return join([# "$(typeof(options.der_use_case))", 
-   # "$(typeof(options.supply_choice_use_case))",
-   "$(typeof(options.der_aggregation_use_case))",
-   "$(typeof(options.market_structure))"],"_")
+    return join([# "$(typeof(options.der_use_case))", 
+            # "$(typeof(options.supply_choice_use_case))",
+            "$(typeof(options.der_aggregation_use_case))",
+            "$(typeof(options.market_structure))"], "_")
 end
 
 
+"""
+    $(TYPEDEF)
+Struct to store the data required for the HEM model.
+$(TYPEDFIELDS)
+"""
 mutable struct HEMData
     # Configuration
-    epsilon::ParamScalar # iteration tolerance
+    "iteration tolerance"
+    epsilon::ParamScalar
```

Comment:
Thank you for these updates!

Reply:
This documentation is fairly redundant with the information in `read_set`, `read_param`. Do you have any ideas about how to write "name" and "description" just once?